src/AzureIoTHub.Portal.Shared/Models/v1.0/DeviceTelemetryDto.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AzureIoTHub.Portal.Shared.Models.v1._0
```

Comment:
Review the namespace here.
```suggestion
namespace AzureIoTHub.Portal.Shared.Models.v10
```